src/sentry/api/endpoints/release_thresholds/health_checks/is_new_issues_count_healthy.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from datetime import datetime
+from logging import getLogger
+
+from sentry import search
+from sentry.api.event_search import SearchFilter, SearchKey, SearchValue
+from sentry.models.project import Project
+from sentry.models.release import Release
+from sentry.models.release_threshold import ReleaseThreshold
+from sentry.models.release_threshold.constants import TriggerType
+from sentry.search.events.constants import RELEASE_ALIAS
+
+default_logger = getLogger("sentry.releases.servicer")
+
+
+def get_new_issue_count_is_healthy(
```

Comment:
It looks like this will fire a query for an individual threshold, is that correct?

Is there any way for us to query for a group of thresholds as opposed to each individually?

Otherwise we may want to think about updating the api to only fetch thresholds for an individual project as opposed to being an index api

Reply:
Nope, each query needs to be done individually. This is echoed by our endpoint count logic as well https://github.com/getsentry/sentry/blob/23.12.1/src/sentry/api/endpoints/organization_issues_count.py#L93-L103

Reply:
Damn :/ 

Ok - let me poke around to see if there's any other ways we can do this too
AFAICT - on the releases page today, we're not making separate API calls for each release, so i would hope there's some way to bulk query this from snuba... but it's possible the api itself is making multiple queries to snuba...